src/pages/Setting/PC/ManageAccount.tsx

```diff
@@ -0,0 +1,140 @@
+import { ReactComponent as ShowIcon } from 'assets/svg/auth/pw-show.svg';
+import { ReactComponent as BlindIcon } from 'assets/svg/auth/pw-blind.svg';
+import { useAuth } from 'store/auth';
+import { ReactComponent as ErrorIcon } from 'assets/svg/auth/error.svg';
+import cn from 'utils/ts/classNames';
+import useTripleBlindCheck from './hook/useTripleBlindCheck';
+import style from './index.module.scss';
+import useModifyPassword, { ERROR_MESSAGE } from './hook/useModifyPassword';
+import PasswordSuccessModal from './PasswordSuccessModal';
+
+export default function ManageAccount() {
+  const {
+    isCurrentBlind,
+    changeCurrentBlind,
+    isNewBlind,
+    changeNewBlind,
+    isNewCheckBlind,
+    changeNewCheckBlind,
+  } = useTripleBlindCheck();
+  const {
+    current,
+    newPassword,
+    check,
+    handleCheck,
+    handleCurrent,
+    handleNewPassword,
+    modifyPassword,
+    isShowError,
+    message,
+    isShowModal,
+  } = useModifyPassword();
+  const auth = useAuth();
+  return (
+    <form className={style.formContainer}>
+
+      <div className={style.title}>계정</div>
+      <div className={style.account}>
+        <span>이메일</span>
+        <input className={style.account__input} value={auth ? auth.email : ''} disabled />
+      </div>
+      <div>
+        <span>아이디</span>
+        <input className={style.account__input} value={auth && 'account' in auth ? auth.account : 'SNSLogin'} disabled />
```

Comment:
요런것도 읽는입장에서는 도메인 지식이 없을 수 있으니 읽기 좋게 함수를 추가하는것도 방법인 것 같아요
```ts
const isEmailLogin = (auth) => 'account' in auth
```
```suggestion
        <input className={style.account__input} value={isEmailLogin(auth) ? auth.account : 'SNSLogin'} disabled />
```

Reply:
46213688d3d4a0aa8cefbb3b44c3de1f4a976812
수정했습니다!